Refactoring the UI for Sign-up Process

sachungo opened the pull request:
[finishes #104704258]
Improved the UI to be more appealing to both restaurateur and diner

sachungo committed:
Refactoring the UI for Sign-up Process

[finishes #104704258]
Improved the UI to be more appealing to both restaurateur and diner

andela-anandaa commented:
Stacey, you will have to pull `develop` branch, and merge it to your branch and do another `push`.

sachungo committed:
Resolved merge conflict in the develop branch

* 'develop' of github.com:andela/resly:
  Fix Search UI [Finishes #104704290] The changes were made so as to improve the search ui hence making search more  appealing
  Fix styleci errors
  Make feature search [Finishes #102342528] This change addresses diner search for restaurants by name
  Write unit test for diner search feature
  Make search feature [Finishes #102342528] This change enables diners search for restaurants by name
  add navbar section
  Fix setup restaurant views The views have been made to use the master layout Also, some emergent issues in the menu-controller were fixed. [finishes #104519006]
  Fix setup restaurant views The views have been made to use the master layout Also, some emergent issues in the menu-controller were fixed.
  add styles section
  Fixed restaurant add view [#104519006]
